### DISCUSSION TITLE
ASCII Text Format of Soundings

### DISCUSSION THREAD

**Discussion by jstndlee** [NOAA-GSL/GSL-Regional-Model-Forum#8] [Q&A] [answered]
Hello,

Is there a URL request parameter that will obtain the ASCII text of a modeled sounding in the format that the interactive Java plot provides it?
I am trying to automate an old piece of software that uses the sounding models, but it was originally written to take in the interactive Java plot text format rather than the GSD format.

V/r,
Justin Lee

**Comment by BillMoninger** [comment, marked as answer]
I'm afraid not. As you mention, it is possible to receive the text of a model sounding in the GSD (now GSL) format, as described at https://rucsoundings.noaa.gov/text_sounding_query_parameters.pdf . But this is not the same as the format used in the text window that appears when you click the 'Get Text' button under the SkewT plot generated when you click the 'Interactive Plot' button at https://rucsoundings.noaa.gov/.
